OpenRA.Mods.Common/Graphics/ContrailRenderable.cs

```diff
@@ -73,11 +73,31 @@ public void Render(WorldRenderer wr)
 			// Start of the first line segment is the tail of the list - don't smooth it.
 			var curPos = trail[Index(next - skip - 1)];
 			var curColor = color;
-			for (var i = 0; i < length - skip - 4; i++)
+
+			for (var i = 1; i < renderLength; i++)
```

Comment:
Why was this changed from 0 to 1?

Reply:
Because `renderLength` is now `length - skip`, for readable I make i = 1 instead of make `i < renderLength - 1`.

If we make i = 0 base on this design, the contrail will be rendered to (0,0) in map.

Reply:
This loop part is tricky and with low readability so I test frequently to get the correct boundary.